IdentityCore/src/controllers/broker/MSIDSSOExtensionInteractiveTokenRequestController.m

```diff
@@ -54,6 +54,8 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
 {
     MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Beginning interactive broker extension flow.");
     
+    [self addQRPINParamIfNeeded];
```

Comment:
Do we have any other places where we modify those parameters today? Maybe there's one that is more appropriate?